pom.xml

```
@@ -1105,6 +1173,7 @@
         <spark321db.version>3.2.1-databricks</spark321db.version>
         <spark322.version>3.2.2</spark322.version>
         <spark330.version>3.3.0</spark330.version>
+        <spark332.version>3.3.2-SNAPSHOT</spark332.version>
         <spark331.version>3.3.1</spark331.version>
```

Nit: Please keep these ordered.
```suggestion
        <spark331.version>3.3.1</spark331.version>
        <spark332.version>3.3.2-SNAPSHOT</spark332.version>
```